### ISSUE TITLE
Game doesn't start (scripthookv_1.0.2060.1) problem

### ISSUE THREAD

**Issue by Thomaatos90** [LukeRoss00/gta5-real-mod#287]
Hi,

I am running into some issues. 
I installed a clean version from steam.
I followed all the instructions. 
But at startup it says fatal error check if you have the latest version of scripthookv.dll. 
Was there an update the last few days of the game or?

Regards,

Thomaatos90

**Comment by LukeRoss00** [comment]
Yes, apparently there was a big update today. I cannot check right now, but it should be possible to revert the game to the previous version by following the instructions [here](https://www.youtube.com/watch?v=nMOa23Yzqss) (at least until Script Hook V gets updated again, which usually takes a few days)

**Comment by Thomaatos90** [comment]
Setup:
Msi creator 17
Rtx 2070 max-q design
I7 10th gen
32gb ram
Oculus rift s

**Comment by Thomaatos90** [comment]
Thnx for the quick response! 
I will try👍

**Comment by Thomaatos90** [comment]
Everything works.
Thank you for the support!

**Comment by LukeRoss00** [comment]
Great! I'll live the issue open until Script Hook V gets updated, so other users can find the workaround easily

**Comment by Ranimate** [comment]
LR, You are so fast with your responses! Thank you so much for everything!
This YouTube link is for Steam version only... I guess I will have to wait with my Epic version

But again, Thank you!

**Comment by celowsky** [comment]
The Quick Setup section of the README.md describes where you can place the old copy of the GTA V files. 

> C:\Program Files (x86)\Steam\steamapps\common\Grand Theft Auto V for the Steam version, C:\Program Files\Epic Games\GTAV for the Epic Games Store version, or C:\Program Files\Rockstar Games\Grand Theft Auto V for the Social Club version.

After making a backup of the files to replaced, I would try to overwrite the newer files with the older ones from the video.

**Comment by Ranimate** [comment]
@celowsky I had tried that, I am getting an error complaining steam_api64.dll could not be found. If I add the dll manually it starts launching but then complains about missing Steam server. So the Epic version would not work for me :(

**Comment by Thomaatos90** [comment]
https://www.youtube.com/watch?v=XyJdfL-Xo0Y <<< try this.

**Comment by Ranimate** [comment]
@Thomaatos90 Thank you my friend, doesnt launch the game, that's ok. I will wait for the right V hook :D

**Comment by Ranimate** [comment]
seems like new ScriptHook is out today - http://www.dev-c.com/gtav/scripthookv/
